docs/readthedocs/usersguide.md

```diff
@@ -1,224 +1,213 @@
+<li>Open a CLI and change directories to the top level directory of the project.</li>
+
+<li>
+Execute ``./gradlew shadowJar`` within the CLI. This creates a fat jar of SIMONA inside of the directory ``build/libs``.</li>
+
+<li>
+For executing a fat jar you need to specify the classpath of the entrypoint of your application. Assuming we are still in the top level directory of our project, the execution command would look as follows:</li>
+</ol>
 
       java -cp build/libs/simona-2.1-all.jar edu.ie3.simona.main.RunSimonaStandalone
 
 
-Using an IDE  
-------------
+### Using an IDE
+
 If you use an integrated development environment (IDE) all you have to do is navigate to the class, and press ``right-click -> run``. The IDE will compile and then execute the code for you.
 
 When executed, the simulation starts at a specified start time and simulates the system interactions until the specified end time.
 As you might wonder who tells SIMONA when to start, when to end and what to actually perform a simulation on, there is indeed a piece missing.
 Since a simulation can be run with all sorts of grids and with different parameter sets, a simulation configuration has to be set for successful execution.
 
-Simulation Inputs
-=================
+## Simulation Inputs
+
+
+### Default Config and Where to Find It
 
-Default Config and Where to Find It
------------------------------------
 
 Within the SIMONA repository, there are two default configurations and their corresponding grids with all necessary data to execute a simulation.
 For each configuration, there is a separate directory in the project directory at ``./inputData``.
 Each configuration is described by a ``.conf`` file that holds the simulation configuration parameters.
 
 
-Set the Simulation Configuration
---------------------------------
-When running a simulation, SIMONA expects the directory of the configuration file as a command-line argument. 
+### Set the Simulation Configuration
+
+When running a simulation, SIMONA expects the directory of the configuration file as a command-line argument.
 Let's pretend you want to start a simulation with the default ``vn_simona`` configuration.
 
-Using a CLI
-^^^^^^^^^^^ 
+#### Using a CLI
+
 When you want to start a simulation run from the command line interface you have to pass the directory of the config file as a command-line argument. That would look as follows:
 
-.. code-block:: none
+    java -cp build/libs/simona-2.1-all.jar edu.ie3.simona.main.RunSimonaStandalone --config=inputData/vn_simona/vn_simona.conf
 
-   java -cp build/libs/simona-2.1-all.jar edu.ie3.simona.main.RunSimonaStandalone --config=inputData/vn_simona/vn_simona.conf
 
+#### Using IntelliJ IDEA
 
-Using IntelliJ IDEA
-^^^^^^^^^^^^^^^^^^^
 
 If you are using IntelliJ IDEA as your IDE, this is how setting the command line argument would look like:
 
+<ol>
+<li> Find the <code>vn_simona.conf</code> file in your project directory and copy its path from the content root </li>
```

Comment:
We've used `` instead of `<code>` everywhere, which looks different when the documentation is compiled. So, I'd suggest using the former syntax here as well.

Reply:
If we use ``, the text won't be shown correctly. I think this is because of the use of <li></li>.

Reply:
@staudtMarius if you use a [different method of creating lists](https://www.markdownguide.org/basic-syntax/#ordered-lists), this works. Keep in mind to use the [right indentation](https://www.markdownguide.org/basic-syntax/#adding-elements-in-lists) for in-between stuff.